sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs

```diff
@@ -86,18 +91,10 @@ public class EventHubProducerClient : IAsyncDisposable
         private EventHubConnection Connection { get; }
 
         /// <summary>
-        ///   An abstracted Event Hub transport-specific producer that is associated with the
-        ///   Event Hub gateway rather than a specific partition; intended to perform delegated operations.
+        ///   A <see cref="TransportProducerPool"/> used to manage a set of partition specific <see cref="TransportProducer"/>.
```

Comment:
@jsquire 

Should ```<see cref="TransportProducerPool"/>``` come with a space before the forward space?

Reply:
For consistency, yes.   😄      In reality, it doesn't matter outside of trigging my formatting OCD.

Reply:
> For consistency, yes. 😄 In reality, it doesn't matter outside of trigging my formatting OCD.

If you ask to me, eventually it will be the sum of those details that will make a difference.

Reply:
If we were in another universe, I would definitely give you a raise.